internal/pkg/api/server/templates/service/cron.go

```diff
@@ -44,14 +44,16 @@ func StartComponentsCron(ctx context.Context, d dependencies.ForServer) error {
 
 	// Start background work
 	go func() {
+		d.Logger().Infof("components cron prepared")
+
 		// Delay start to a rounded time
 		interval := ComponentsUpdateInterval
 		startAt := time.Now().Truncate(interval).Add(interval)
 		timer := time.NewTimer(time.Until(startAt))
 		<-timer.C
 
 		// Start ticker
-		d.Logger().Infof("components updater started at %s, interval=%s", time.Now().Format("15:04:05"), interval)
+		d.Logger().Infof("components cron started at %s, interval=%s", time.Now().Format("15:04:05"), interval)
```

Comment:
Upravil som hlasky, aby bol jasnejsie odliseny cron od konkretneho updatu (`updater` vs `update`) + cele som to aj manualne vyskusal:

![image](https://user-images.githubusercontent.com/19371734/188077988-f2e377d7-5288-4cbe-ac27-603d42d2d617.png)